arena.js

```diff
@@ -0,0 +1,35 @@
+class Arena {
+  constructor(name){
+    this.name=name.charAt(0).toUpperCase() + name.slice(1);
+    this.gladiators=[];
+    console.log(" Welcome to "+name+"!!!")
+  }
+  addGladiator(name){
+      this.gladiators.push(name);
+      console.log(this.gladiators+" added!!!")
+      console.log(this.gladiators.length);
+
+    if (this.gladiators.length==2){
+      return "You already have two players";
+    }
+  }
+
+fight(){
+  if (this.gladiators.length===2){
+    if (this.gladiators[0]==this.gladiators[1]){
```

Comment:
You should be comparing the gladiators' weapons -- not the gladiators themselves. Otherwise, this condition will never trigger unless they have the same weapon **AND** name.